iac-extensions/cloudformation/src/main/java/org/sonar/iac/cloudformation/checks/DisabledDBEncryptionCheck.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * SonarQube IaC Plugin
+ * Copyright (C) 2021-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.iac.cloudformation.checks;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.sonar.check.Rule;
+import org.sonar.iac.common.api.checks.CheckContext;
+import org.sonar.iac.common.api.checks.SecondaryLocation;
+import org.sonar.iac.common.api.tree.PropertyTree;
+import org.sonar.iac.common.checks.PropertyUtils;
+import org.sonar.iac.common.checks.TextUtils;
+
+@Rule(key = "S6303")
+public class DisabledDBEncryptionCheck extends AbstractResourceCheck {
+
+  private static final String MESSAGE_RDS_DB_INSTANCE = "Make sure that using unencrypted RDS DB Instances is safe here.";
+  private static final String MESSAGE_RDS_CLUSTER = "Make sure that using an unencrypted RDS DB Cluster is safe here.";
+  private static final String MESSAGE_RDS_DB_GLOBAL_CLUSTER = "Make sure that using an unencrypted RDS DB GlobalCluster is safe here.";
+
+  private static final String OMITTING_MESSAGE = "Omitting \"StorageEncrypted\" disables databases encryption. Make sure it is safe here.";
+  private static final String SECONDARY_MESSAGE = "Related RDS DBInstance";
+  private static final List<String> EXCLUDE_AURORA_ATTRIBUTE = List.of("aurora", "aurora-mysql", "aurora-postgresql");
+  private static final String KEY_STORAGE_ENCRYPTED = "StorageEncrypted";
+
+  @Override
+  protected void checkResource(CheckContext ctx, Resource resource) {
+    if (!(resource.isType("AWS::RDS::DBInstance")
```

Comment:
Small addition: let's maybe align this with how it is done in other Cloudformation checks (no early return, and just `if, else if, else if`). See `WeakSSLProtocolCheck` and `DisabledLoggingCheck` for example.

Reply:
done